resources/js/views/settings/roles/View.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <div>
+    <h3>
+      {{ id === 'new' ? $t('settings.roles.new') : (
+        viewOnly ? $t('settings.roles.view', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+          : $t('settings.roles.edit', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+      ) }}
+    </h3>
+    <hr>
+
+    <b-form @submit='saveRole'>
+      <b-container fluid>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='name'>{{ $t('settings.roles.name') }}</label>
+          </b-col>
+          <b-col sm='9'>
+            <b-form-input :state='fieldState("name")' id='name' type='text' v-model='model.name' :disabled='isBusy || viewOnly'></b-form-input>
+            <b-form-invalid-feedback>
+              {{ fieldError('name') }}
+            </b-form-invalid-feedback>
+          </b-col>
+        </b-row>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='room-limit'>{{ $t('settings.roles.roomLimit') }}</label>
```

Comment:
Please add checkboxes to set room-limit -1 = unlimited, or room-limit = null (system default)